actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -17,6 +17,7 @@ class Railtie < Rails::Railtie # :nodoc:
     config.action_dispatch.encrypted_cookie_salt = 'encrypted cookie'
     config.action_dispatch.encrypted_signed_cookie_salt = 'signed encrypted cookie'
     config.action_dispatch.perform_deep_munge = true
+    config.action_dispatch.access_control_headers = { }
```

Comment:
`access_control_headers` will directly be passed into `Rack::File`, so the name doesn't necessarily have to be prefixed by `access_control_`. But if we change this name to just `asset_headers` or `static_file_headers`, then we also should deprecate `static_cache_control` config in favor of it.